sdk/go/pulumi-language-go/main.go

```diff
@@ -811,6 +812,10 @@ func debugCommand(bin string) (*exec.Cmd, *debugger, error) {
 	contract.IgnoreClose(logFile)
 	args := []string{"--headless=true", "--api-version=2"}
 	args = append(args, "--log", "--log-dest", logFile.Name())
+	port, err := netutil.FindNextAvailablePort(57134)
```

Comment:
Nit: Would it be worth defining the port as a const at the top of the file, so it's more obvious to see rather than "buried" in the implementation?

```go
const preferredDebugPort = 57134
```

or something like that.


Same for python.

Reply:
Agreed, that's nicer :heavy_check_mark: